packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -419,6 +548,51 @@ class _ContrastReport {
   }
 }
 
+/// Gives the colors of all pixels inside a given rectangle on the screen.
+///
+/// Given a [ByteData] object [data], which stores the color of each pixel
+/// in row-first order, where each pixel is given in 4 bytes in RGBA order,
+/// and [paintBounds], the rectangle,
+/// and [width] and [height], the dimensions of the [ByteData],
+/// returns a list of the colors of all pixels within the rectangle in
+/// row-first order.
+///
+/// In the returned list, each color is represented as a 32-bit integer
+/// in ARGB format, similar to the parameter for the [Color] constructor.
+List<int> _subsetFromRect(ByteData data, Rect paintBounds, int width, int height) {
```

Comment:
This seems like a better name than _subsetToRect however something like _uniqueColorsWithinRect might be even more evocative of the function's purpose.

Reply:
Changed to `_colorsWithinRect`.